libsolutil/CDCL.h

```diff
@@ -113,7 +174,22 @@ class CDCL
 	std::vector<size_t> m_decisionPoints;
 	/// Index into assignmentTrail: All assignments starting there have not yet been propagated.
 	size_t m_assignmentQueuePointer = 0;
+
+	// Current state of the solver. If FALSE, we are in an UNSAT state.
+	bool ok = true;
```

Comment:
```suggestion
	bool m_ok = true;
```
(although I also find "ok" not descriptive enough)

Reply:
Good pointer, actually Fixing to:

```
enum class State {unknown, unsat};
State m_state = State::unknown;
```

Which I think captures it much better?

Reply:
Sounds good!

Reply:
Thanks :)